feature-libs/cart/saved-cart/events/events/saved-cart-event.builder.ts

```diff
@@ -144,22 +138,21 @@ export class SavedCartEventBuilder {
    * @returns
    */
   protected buildRestoreSavedCartEvents<T>(
-    mapping: ActionToEventMapping<T>,
-    saveTime?: boolean
+    mapping: ActionToEventMapping<T>
   ): () => void {
     const eventStream$ = this.getAction(mapping.action).pipe(
-      switchMap((action) => {
-        return of(action).pipe(
+      switchMap((action) =>
+        of(action).pipe(
           withLatestFrom(this.multiCartService.getCart(action.payload.cartId))
-        );
-      }),
+        )
+      ),
       map(([action, cart]) =>
         createFrom(mapping.event as Type<T>, {
           ...action.payload,
           cartCode: cart?.code,
           saveCartName: cart?.name,
           saveCartDescription: cart?.description,
-          ...(saveTime && { saveTime: cart?.saveTime }),
+          ...(cart?.saveTime && { saveTime: cart?.saveTime }),
```

Comment:
Can you try to do just `...cart?.saveTime,`?
I'm expecting to set the `saveTime: xxxxxx` if the `cart?.saveTime` is present. Otherwise, this property should not exist in the payload.

Reply:
Nvm, started discussion offline